assets/components/checkoutCopy/checkoutCopy.jsx

```diff
@@ -0,0 +1,30 @@
+// @flow
+
+// ----- Imports ----- //
+
+import React from 'react';
```

Comment:
can we use `ProductPageTextBlock` instead? going from the CSS and props they feel similar enough but I'm not sure without screenshots

Reply:
As far as I understand it, that component is for headings and content in the sections of the product page? The `CheckoutCopy` component is used for the chunks of text just above the button in the screenshots above.

Reply:
ah, I can see it now! `TextBlock` is only concerned with different types of content under a heading and caps out the max width. It's close but not the same! let's leave this like that (with `gu-span` if possible!